src/main/java/org/nikita/chiken_bell/core/exception/PhoneExistException.java

```diff
@@ -0,0 +1,4 @@
+package org.nikita.chiken_bell.core.exception;
+
+public class PhoneExistException extends RuntimeException {
```

Comment:
Change name to "PhoneAlreadyExistException"

Reply:
Ok